code/src/index.css

```diff
@@ -11,3 +18,124 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+#root {
+  display: flex;
+  justify-content: center;
+}
+
+.form-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 300px;
+}
+
+form {
+  border: 1px solid #000;
+  box-shadow: 5px 6px #000;
+  padding: 10px;
+  background-color: #f2f0f0;
+  font-family: "Manlo", monospace;
+  font-weight: bold;
+  font-size: 12px;
+  margin-top: 30px;
+  margin-bottom: 30px;
+}
+
+.text-input {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-family: "Inconsolata", monospace;
+  padding: 10px;
+  border: 1px solid #a4a4a4;
+  box-sizing: border-box;
+  resize: none;
+}
+
+.send-button {
+  display: flex;
+  align-items: center;
+  height: auto;
+  border-top-left-radius: 17px;
+  border-bottom-left-radius: 17px;
+  border-top-right-radius: 17px;
+  border-bottom-right-radius: 17px;
```

Comment:
since you have the same radius (17px) on all corners, you can simply write **border-radius: 17px;** to save time and space :)